device/tt_silicon_driver.cpp

```diff
@@ -90,7 +64,7 @@ const uint32_t HUGEPAGE_MAP_MASK = HUGEPAGE_REGION_SIZE - 1;
 static const uint32_t MSG_ERROR_REPLY = 0xFFFFFFFF;
 
 // Hardcode (but allow override) of path now, to support environments with other 1GB hugepage mounts not for runtime.
-const char* hugepage_dir_env = std::getenv("TT_BACKEND_HUGEPAGE_DIR");
+const char* hugepage_dir_env = std::getenv("TT_UMD_HUGEPAGE_DIR");
```

Comment:
If you're changing this, you need to make sure that you're not breaking some pipeline/setup somewhere. For instance, maybe the cloud team relies on setting TT_BACKEND_HUGEPAGE_DIR env?

Reply:
Reverted this, created the issue to unify naming of all env vars https://github.com/tenstorrent/tt-umd/issues/146